src/Core/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## NOT RELEASED
 
+- Athena Client: Documentation updates.
```

Comment:
Should be removed, the documentation is not part of the component

(same for other readme files)